src/containers/Debug/fusionChartDemo.js

```diff
@@ -0,0 +1,256 @@
+import React, { Fragment } from "react";
+import { connect } from "react-redux";
+import { compose } from "redux";
+
+import { firebaseConnect } from "react-redux-firebase";
+
+// Charting packages from fusionChart
+import FusionCharts from 'fusioncharts';
+import Charts from 'fusioncharts/fusioncharts.charts';
+import powerCharts from 'fusioncharts/fusioncharts.powercharts';
+import ReactFC from 'react-fusioncharts';
+import FusionTheme from 'fusioncharts/themes/fusioncharts.theme.fusion';
+
+import Button from '@material-ui/core/Button';
+import { Typography } from "@material-ui/core";
+
+ReactFC.fcRoot(FusionCharts, Charts,powerCharts, FusionTheme);
+
+
+class fusionChartDemo extends React.PureComponent {
+    constructor(props) {
```

Comment:
hi, three suggestions:
1. React class already provide the `props`, and you can initialize the state by calling `state = {}` as a class property.
2. Can you follow the indentation of 2 spaces like the rest of the codebase
3. This class component can be in a different `container` folder, on the same level as `Debug`

Reply:
Ok, noted!